shared/nm-utils/nm-dedup-multi.c

```diff
@@ -994,6 +994,64 @@ nm_dedup_multi_objs_to_ptr_array_head (const NMDedupMultiHeadEntry *head_entry,
 	return result;
 }
 
+/**
+ * nm_dedup_multi_entry_reorder:
+ * @entry: the entry to reorder. It must not be NULL (and tracked in an index).
+ * @entry_order: (allow-none): an optional other entry. It MUST be in the same
+ *   list as entry. If given, @entry will be ordered after/before @entry_order.
+ *   If left at %NULL, @entry will be moved to the front/end of the list.
+ * @order_after: if @entry_order is given then %TRUE means to order @entry after
+ *   @entry_order (otherwise before). If @entry_order is %NULL, %TRUE means to
+ *   move @entry to the end of the list (otherwise the beginning).
+ *   Note that @order_after %TRUE, means to link @entry after @entry_order,
+ *   but it means to link @entry before the list's head (to become the tail).
```

Comment:
Well, not very clear to me what this last sentence mean... if @order_after is TRUE, then @entry will never be linked before the list's HEAD, isn't it?